air/src/air/context.rs

```diff
@@ -237,6 +230,50 @@ impl<B: StarkField> AirContext<B> {
         self.num_transition_exemptions
     }
 
+    /// Returns the number of columns needed to store the constraint composition polynomial.
+    /// This is the maximum of:
+    ///
+    /// 1. The maximum evaluation degree over all transition constraints minus the degree
```

Comment:
minor nit: I'd put the blank line between the first two lines, like so:
```
/// Returns the number of columns needed to store the constraint composition polynomial.
///
/// This is the maximum of:
/// 1. The maximum evaluation degree over all transition constraints minus the degree
```
The reasons is that the first line will be used as the method summary in the docs and if we don't have this blank line "This is the maximum of:" would be included in the method summary and it will look incomplete.

Reply:
Thanks!
I have fixed it now